src/main/include/subsystems/Intake.h

```diff
@@ -0,0 +1,24 @@
+//uses sparkmax for now
+#pragma once
+#include <rev/SparkMax.h>
+#include <frc2/command/SubsystemBase.h>
+#include <frc2/command/InstantCommand.h>
+#include <rev/config/SparkMaxConfig.h>
+using namespace rev::spark;
+
+namespace t34{
+    
+    class Intake : public frc2::SubsystemBase {
+        public:
+        
+        Intake();
+
+        frc2::CommandPtr RunIn(double power_percentage);
+        frc2::CommandPtr RunOut(double power_percentage);
```

Comment:
In FIRST's online examples, they always add a `Command` suffix to calls likes these:

```suggestion
        frc2::CommandPtr RunInCommand(double power_percentage);
        frc2::CommandPtr RunOutCommand(double power_percentage);
```

Reply:
Another alternative nomenclature in case you like it better:

```suggestion
        frc2::CommandPtr TakeInCommand(double power_percentage);
        frc2::CommandPtr SpitOutCommand(double power_percentage);
```

Reply:
Took a vote on Feb 1 and decided to change the names to AlgaeInCommand and AlgaeOutCommand